data/spells/scripts/support/food.lua

```diff
@@ -1,19 +1,19 @@
-local food = {
+local foods = {
 	2666, -- meat
 	2671, -- ham
 	2681, -- grape
 	2674, -- apple
 	2689, -- bread
 	2690, -- roll
-	2696  -- cheese
+	2696 -- cheese
 }
 
 function onCastSpell(creature, variant)
-	creature:addItem(food[math.random(#food)])
-	if math.random(1, 100) > 50 then
-		creature:addItem(food[math.random(#food)])
+	if math.random(1, 100) >= 50 then
```

Comment:
Oh yeah, I'm more used to working with 0-based indexes and ranges and Lua patterns still confuse me :P